docs/source/release_notes.rst

```diff
@@ -0,0 +1,31 @@
+Release Notes
+=============
+
+.. role:: small
+
+Version 1.0
+-----------
+
+1.0.2 :small:`2021-03-26`
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. rubric:: Bugfixes
+
+- Fix not catching ``ArpackError`` when computing stationary distribution and mypy-related linting issues
+  `PR 21 <https://github.com/msmdev/pyGPCCA/pull/21>`_.
+
+.. rubric:: Improvements
+
+- Use PETSc/SLEPc, if installed, to compute stationary distribution speed it up
```

Comment:
Please correct grammar, e.g. into "... to speed up the computation of the stationary distribution"

Reply:
Thanks for spotting this, itis fixed now.